docs/_pages/collections.md

```diff
@@ -41,6 +41,9 @@ collection.Should().EndWith(8);
 collection.Should().EndWith(new[] { 5, 8 });
 
 collection.Should().BeSubsetOf(new[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, });
+collection.Should().BeProperSubsetOf(new[] { 1, 2, 5, 6, 7, 8 });
```

Comment:
🤔 I would some comments here to explain the difference (again) between a normal subset/superset and a proper one.